roles/ucp/templates/ucp.j2

```diff
@@ -40,12 +40,17 @@ start)
     {% else -%}
     /usr/bin/docker run --rm -t --name ucp \
         -v /var/run/docker.sock:/var/run/docker.sock \
-        -e UCP_ADMIN_USER="admin" -e UCP_ADMIN_PASSWORD="orca" \
+        -e UCP_ADMIN_USER={{ ucp_admin_user }} -e UCP_ADMIN_PASSWORD={{ ucp_admin_password }} \
         docker/ucp join --host-address={{ node_addr }} \
           --image-version={{ ucp_version }} \
           --url="https://{{ service_vip }}:443" \
+    {% if run_as is defined and run_as == "master" -%}
```

Comment:
@vvb sorry to nit pick this but we should avoid this check `run_as is defined`.

It is good to fail if this variable is not defined than treating it to be `worker` as we depend on this variable at a lot of places and a check like this might result in introduction of wrong behavior and harder to debug situations.